app/javascript/components/map/component.jsx

```diff
@@ -261,45 +281,71 @@ class MapComponent extends Component {
         className={cx('c-map', { 'no-pointer-events': drawing }, className)}
         style={{ backgroundColor: basemap && basemap.color }}
       >
-        <Map
-          mapStyle={mapStyle}
-          viewport={viewport}
-          bounds={this.state.bounds}
-          onViewportChange={this.onViewportChange}
-          onClick={this.onClick}
-          onLoad={this.onLoad}
-          interactiveLayerIds={interactiveLayerIds}
-          attributionControl={false}
-          minZoom={minZoom}
-          maxZoom={maxZoom}
+        <Tooltip
+          theme="tip"
+          title="GFW Interactive Map"
+          hideOnClick={false}
+          html={
+            <Tip
+              text={
+                this.state.clicks > 0
+                  ? 'Click to add a point or close shape.'
```

Comment:
I think we still need an intermediate state here for `click > 0 && clicks < 3` which says `click to add another point`